src/python/WMComponent/DBS3Buffer/DBSUploadPoller.py

```diff
@@ -94,34 +94,19 @@ def uploadWorker(workInput, results, dbsUrl, gzipEncoding=False):
             dbsApi.insertBulkBlock(blockDump=block)
             results.put({'name': name, 'success': "uploaded"})
         except HTTPError as ex:
-            # when we will have new dbs3-client and fine granularity DBS Go server
-            # error codes then we should remove if/elif/else and replace it with
-            # DBSError(ex.body), e.g.
-            # dbsError = DBSError(ex.body)
-            # reason = dbsError.getReason()
-            # srvCode = dbsError.getServverCode()
-            # results.put({'name': name, 'success': 'error', 'error': reason, 'server_code': srvCode})
-
-            # legacy code which deal with DBS python server exceptions
             dbsError = DBSError(ex.body)
             reason = dbsError.getReason()
-            if 'Block %s already exists' % name in reason:
-                # Then this is probably a duplicate
-                # Ignore this for now
+            message = dbsError.getMessage()
+            srvCode = dbsError.getServerCode()
+            msg = f'DBSError: {message}, reason: {reason}'
```

Comment:
maybe add the server code to this variable as well, e.g.:
`msg = f'DBSError error code: {srvCode}, message: {message}, reason: {reason}'`

Reply:
done